app/models/friendship.rb

```diff
@@ -1,4 +1,8 @@
 class Friendship < ApplicationRecord
   belongs_to :user
   belongs_to :friend, class_name: 'User'
+
+  def mutual
+    user.friends & friend.friends
+  end
```

Comment:
Nice start!! But the purpose of this milestone is to actually use the method!! In this case, the method is not actually creating the double friendship, but just making a collection of the user's friends and the friend's friends which will be unnecessary once you create the double row friendship.

Please check the comments I left in the main message.